compiler/rustc_target/src/spec/targets/sparc_unknown_none_elf.rs

```diff
@@ -7,7 +7,7 @@ pub(crate) fn target() -> Target {
         linker: Some("sparc-elf-gcc".into()),
         endian: Endian::Big,
         cpu: "v7".into(),
-        abi: "elf".into(),
+        abi: "".into(),
```

Comment:
I'd suggest you just get rid of this line entirely, since I think you'll inherit the default provided via `Default::default()` below which (IIUC) will also be `abi: ""`